app/common/common-module.js

```diff
@@ -25,8 +25,7 @@ angular.module('ushahidi.common', [
 .service('MediaEndpoint', require('./services/endpoints/MediaEndpoint.js'))
 .service('MessageEndpoint', require('./services/endpoints/message.js'))
 
-.service('PostViewHelper', require('./services/view-helper.js'))
-.service('ModeViewHelper', require('./services/mode-view-helper.js'))
```

Comment:
I've dropped the extra helper. @middle8media feel free to modify existing services if needed, don't duplicate them.
We can always role things back in git.